Trippy/Query Handlers/UserAuthHandler.h

```diff
@@ -6,6 +6,7 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "AFNetworking.h"
```

Comment:
One philosophy is to keep your headers as clean as possible, even to the point of forward declaring classes (for example,
```
@class PFObject;
```
). That way, .m files that care about the internals of classes can include them, but no one else has to. It should improve compile times as well.